packages/shared/src/forms/datePicker/DatePicker.ios.js

```diff
@@ -82,7 +90,7 @@ export default class DatePicker extends React.Component<Props, State> {
             {...rest}
             minimumDate={minDate}
             maximumDate={maxDate}
-            date={this.state.date}
+            date={this.state.date || new Date()}
```

Comment:
Same comment; are these default values necessary?

Reply:
Yes, if this.state.date is null, then DatePickerIOS will crash the entire app. Which is what happend on my demo on monday 😞 

And yes, it should be `DateUtils.getUTCToday()`